purchases/src/main/kotlin/com/revenuecat/purchases/interfaces/PurchaseCallback.kt

```diff
@@ -11,3 +14,34 @@ interface PurchaseCallback : PurchaseErrorCallback {
      */
     fun onCompleted(storeTransaction: StoreTransaction, customerInfo: CustomerInfo)
 }
+
+// TODO rename, update docs
+interface NewPurchaseCallback : PurchaseErrorCallback {
+    /**
+     * Will be called after the purchase has completed
+     * @param storeTransaction StoreTransaction object for the purchased product.
+     * @param customerInfo Updated [CustomerInfo].
+     */
+    fun onCompleted(storeTransaction: StoreTransaction?, customerInfo: CustomerInfo)
```

Comment:
`StoreTransaction` is nullable here because this could be deferred or something, right? Do we need to update the docs here explaining that?

Reply:
yep 💯 definitely should